katello-agent/katello-agent.spec

```diff
@@ -39,15 +40,35 @@ Requires: yum-plugin-security
 Requires: yum-security
 %endif
 
+%description
+Provides plugin for gofer, which allows communicating with Katello server
+and execute scheduled actions.
+
+%package -n katello-host-tools
+BuildArch: noarch
+Summary: A set of commands and yum plugins that support a katello host
+
+Requires: subscription-manager
+Requires: %{name}-fact-plugin
```

Comment:
Will this result in `katello-agent-fact-plugin` or the subpackage name being used? `katello-host-tools-fact-plugin` ?